circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  pre:
+    - cd /opt/circleci/.pyenv/plugins/python-build/../.. && git pull && cd -
+  python:
+    version: 3.5.5
+  timezone:
+    America/Los_Angeles
+
+dependencies:
+  override:
+    - pip install -U pip wheel
+    - make
+    - pyenv local 2.7.11 3.5.5
```

Comment:
Do we want to support 2.7? I suggest we only support Py3

Reply:
We can remove. Olekisy added back in `tox.ini` and I fixed the broken CI

Reply:
I'm suggesting we don't support 2.7 at all.